tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
```

Comment:
Since this library has an Async component, using ES2017 (with async await) as a target might be better.

Reply:
https://github.com/bitcoinjs/bip39/pull/104/commits/ac0da78a1b3212328299dc4e5e65b6a077f71246